### DISCUSSION TITLE
in the document of Spark Doris Connector, the param "sink.batch.size" seems not correct

### DISCUSSION THREAD

**Discussion by hwanginsitein** [apache/doris#25289] [A - General / Q&A]
in the code **_ConfigurationOptions.java_**
I only get the param **_doris.sink.batch.size_**
![333333](https://github.com/apache/doris/assets/7668555/cd8d4427-8a40-458e-8518-9c0427fafce4)

anyone can explains?

**Comment by yagagagaga** [comment]
Your image seems to be unable to load.
